### ISSUE TITLE
Decoration is still information, of course ... Guideline 2.1.1 Image alternatives

### ISSUE THREAD

**Issue by brett-oppegaard** [w3c/wcag3#369]
I am perplexed, and have been for a long time, how "decorative" imagery in WCAG is defined and discounted and dismissed as non-informational. 

I was having a discussion last week with a web designer who was claiming that any stock photo, like a diverse group of people in a grassy field, reading books, in front of a university building, was "decorative" and therefore not required by law to be described. I have heard other people say portrait photos, of say, organizational leaders, are "decorative," plus many other similar claims, and of course, when a decorative element is used, like a fleur-de-lis, or whatever, a visual designer still chose to include that image in the interface for some reason, right? Is there a connection to French culture? Is there some reference to a lily or iris? Why was it placed there? Since visual designers ALWAYS have the option to include or not include imagery, and when they do choose to include it, it always has some rhetorical purpose, I think this "decorative" loophole in WCAG needs to be eliminated. 

No. 1, what is the purpose of this "decorative" exemption? If that answer is not audience-oriented, which I'm assuming it's not, then is WCAG for the ease and convenience of the designers, or for the inclusion of the audience members? 

No. 2., if people who are blind are excluded from learning about all of these "decorative" elements on websites and mobile apps, how will people who cannot see or cannot see well be included fully in societal discussions? 

No. 3, what is the downside of eliminating this loophole? More Audio Description for websites, more of a focus on inclusivity for blind people, more data and content for listeners and readers? I just don't understand in any way why this exemption should exist. My assumption is that it provides a loophole for computer scientists, web designers, and content creators to avoid tough decisions about their workflow and responsibilities to diverse audiences. What else could it be? 

No. 4, maybe a long time ago, a bit of extra screenreader text could have microscopically slowed down a webpage page loading, but those days are long gone. So is this a technical issue, and if so, is this a contemporary (or antiquated) technical issue? 

No. 5, already inferred, but if "decorative" is going to be kept as a loophole to avoid full media accessibility, then WCAG is obligated to clearly define "decorative" in a legal sense, and also justify why decorative imagery is not information that also should be shared with all audience members.

Hope that helps!

Commenting on the following:

2.1 Image and media alternativesDeveloping
Guideline 2.1.1 Image alternatives
Users have equivalent alternatives for images.

Which foundational requirements apply?
Foundational requirement: Decorative imageDeveloping
[Decorative](https://www.w3.org/TR/wcag-3.0/#dfn-decorative) [images](https://www.w3.org/TR/wcag-3.0/#dfn-image) are [programmatically](https://www.w3.org/TR/wcag-3.0/#dfn-programmatically-determinable) hidden.

**Comment by TestPartners** [comment]
This topic arises almost every week on one accessibility forum or another, and there is no "right" answer. Furthermore, opinion has changed over the last 20 years or so. Back then, there was pretty much universal acceptance that text alternatives for images should just be the information that is necessary for understanding the content. Almost no one would have said that text alternatives are required for decorative images.

However, in recent years, some people have started to say that they like decorative images to be described because they convey the feel of the website even if they do not provide information. The problem is that this is a serious impediment to screen reader users who do not want to hear all that fluff, and they are annoyed that it slows them down. Anecdotally, I would say that opinion is split about 50:50, and I would say there should be a substantial majority one way or the other before changing anything.

We now have ridiculous situations where content creators are over-compensating by writing lengthy literal descriptions of decorative images. It's not uncommon for these to comprise several sentences totalling as much as 50 words. It can be necessary for a screen reader user to listen to a large proportion of this before deciding it's fluff they don't need to listen to.

I recently suggested that a new CSS "prefers-reduced-fluff" property should be introduced so users can choose whether to hear the descriptions of decorative images. I was only partially joking but I don't think there is any other way to satisfy both camps.

FWIW, I don't accept the premise that undescribed decorative images discriminate against blind people in some way. You may as well argue that painting our walls any colour other than white, or hanging wallpaper discriminates against people who can't see it. Would you have us all wear the same colour clothes?

**Comment by TestPartners** [comment]
Any definitions must acknowledge that providing too much information can be as much of an accessibility barrier as providing too little, and it may arguably be worse.

**Comment by brett-oppegaard** [comment]
Thanks for your comments; I appreciate your feedback, which crosses into
territories that I do hear about quite a bit, so here are my thoughts on
the matters.

*Important points to address:*


   - "This topic arises almost every week on one accessibility forum or
   another, and there is no 'right' answer." And opinions are changing in
   favor of more description. ... Yes, therefore, *an urgent (meaning in
   WCAG3, not pushed back to WCAG4), thoughtful, and thorough response is
   needed*, as you are contending here, to a topic that is arising *all
   over, and all of the time on accessibility forums*.
   - "The problem is that this is a serious impediment to screen reader
   users who do not want to hear all that fluff," ... I disagree with the
   "serious impediment" assertion; all it takes to move on to the next piece
   of content is a single press of the down-arrow button (or whatever
   keystroke/macro is set up for that user) on the screenreader. Fluff? *If
   half the people you anecdotally are imagining want it, then I wouldn't
   consider that "fluff." **Who wants to totally disregard half of their
   audience?*
   - "where content creators are over-compensating by writing lengthy
   literal descriptions of decorative images." ... that seems to me *the
   spirit of WCAG2's "robust," right?* And 50 words is lengthy? I guess we
   could debate length, efficiency, and efficacy, but description takes what
   it takes, and as I wrote earlier, the easy solution to someone who doesn't
   want it, or has had enough, one button push to move on to the next piece of
   content.
   - "I don't think there is any other way to satisfy both camps." ...
   There is, actually: Provide the robust content, per WCAG guidelines, and
   create usable screenreader systems, coding schemas, and writing
   templates/training that allow people to make navigable content and to
   easily navigate to what they want to listen to, and for how long. *Why
   should people who want less information be privileged over people who want
   more, when the less crowd and more crowd can be both satisfied with the
   same text? *
   - "I don't accept the premise that undescribed decorative images
   discriminate against blind people in some way. You may as well argue that
   painting our walls any colour other than white, or hanging wallpaper
   discriminates against people who can't see it. Would you have us all wear
   the same colour clothes?" ... No; but how about people routinely get in the
   habit of describing their physical surroundings (wall colors) and fashion
   choices in situations where people in the audience might have trouble
   seeing or seeing well?* It really doesn't take that much more time or
   effort.* ... This is starting to shift out of web compliance into
   accessible physical environments and social situations, which is another
   major issue to address, but let's try to return to WCAG.

**Comment by yatil** [comment]
> This topic arises almost every week on one accessibility forum or another, and there is no "right" answer. Furthermore, opinion has changed over the last 20 years or so. Back then, there was pretty much universal acceptance that text alternatives for images should just be the information that is necessary for understanding the content. Almost no one would have said that text alternatives are required for decorative images.

WCAG said literally that there is only an exception for images that are “pure decoration” since 2008. Just being mildly decorative never qualified for the exception. WCAG comes from when you had to use images to render shadows for containers. When you used image replacement techniques for headings. You cannot argue that the image of a person in an index of personell is “pure decoration” or that news images are.

> However, in recent years, some people have started to say that they like decorative images to be described because they convey the feel of the website even if they do not provide information. 

If they add to the meaning of the page, they should have alternative text. People wanted an easy way out of describing images and just said “everything is decorative”. But decorative is not “pure decoration”.

> The problem is that this is a serious impediment to screen reader users who do not want to hear all that fluff, and they are annoyed that it slows them down. Anecdotally, I would say that opinion is split about 50:50, and I would say there should be a substantial majority one way or the other before changing anything.

Yes, some people do too much, and in some situations it’s too much. This can happen.

> We now have ridiculous situations where content creators are over-compensating by writing lengthy literal descriptions of decorative images. It's not uncommon for these to comprise several sentences totalling as much as 50 words. It can be necessary for a screen reader user to listen to a large proportion of this before deciding it's fluff they don't need to listen to.

Again, this is not a requirement, and people might overcorrect. I have not seen this, I still see the majority of images undescribed. The WCAG 3 wording forbids alternative text for every image that does not contribute to the understanding of the page at all. Even if it would be interesting or good context.

The rest of your comment is irrelevant to this issue.

**Comment by brett-oppegaard** [comment]
I think the WCAG aim is "equivalent" information, not too much, not too
little; in the Goldilocks zone. At this point, I think it is very difficult
to argue that any audience member is getting "too much" information,
holistically, by trying to use the web with a screenreader, at least in
comparison to the individuals using the same web pages through sight and a
visual interface with the same informational needs; in other words, in an
equivalent state of need and use. So I think this argument is a red
herring. It's sort of like saying, there are way too many books in the
library, way too many shows on television, way too many movies in the
theater; novels are too long; and so on. The simple answer: When you, as
the consumer of the information, have enough to satisfy you, do something
else.

If you flip this idea on its head, the argument becomes ridiculous to
suggest that sighted people have too many photos, too many videos, too many
maps, too much visual media everywhere all of the time, so the answer is to
somehow and in some way arbitrarily decide for everyone what the *right*
amount of information is. Don't you really think the audience should
decide, not the WCAG board?

**Comment by lauracarlson** [comment]
Related: https://github.com/w3c/wcag3/issues/238
